File: OJ/2학기/PythonProgramming/ch.3/3-2.py
'''아래 input함수를 사용하여 문자열을 2개받고, int로 변환하라.
int값들의 float 나누기, int값들의 int 나누기 값을 입출력 예시를 따라 출력하라.
작은 따옴표, 큰 따옴표안에도 출력해보자.
sa = input('enter int a: ') #sa는 문자열로 입력된다.
sb = input('enter int b: ') #sb는 문자열로 입력된다.'''

sa = input('enter int a: ')
sb = input('enter int b: ')
sa = int(sa) #int변환
sb = int(sb)
print(f"{sa}/{sb}={sa/sb}")
print(f"{sa}//{sb}={sa//sb}")
print(f"\'{sa}/{sb}={sa/sb}\'")
print(f"\"{sa}//{sb}={sa//sb}\"")

File: OJ/2학기/심화프로그래밍/ch.3/3-2.c
/*2개의 숫자를 입력받아, 조건 연산자를 활용하여 최대값을 출력하는 프로그램을 작성해보자.
[ 조건 연산자: 조건식 ? 참일 경우 : 거짓일 경우 ]

# 입출력 예시
Input >> 10 2
Max is 10.
*/

#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>

int main(){
    int a, b, max;

    printf("Input >> ");
    scanf("%d %d", &a, &b);

    max = (a > b) ? a : b;

    printf("Max is %d.\n", max);

    return 0;
}